common-treble.mk

```diff
@@ -67,6 +67,10 @@ PRODUCT_PACKAGES += \
     android.hardware.camera.provider@2.4-service
 endif
 
+# Sensor service
```

Comment:
```suggestion
# Camera
```
Just like above.

I don't think this is used by the "sensors" (light, gyro, accel etc), or is it?

Reply:
maybe multihal is using this, I have no way to check